lnbits/static/js/components/payment-list.js

```diff
@@ -158,6 +158,14 @@ Vue.component('payment-list', {
       const params = new URLSearchParams(query)
       LNbits.api.getPayments(this.wallet, params).then(response => {
         const payments = response.data.data.map(LNbits.map.payment)
+        if (detailed) {
+          this.paymentsCSV.columns.push({
+            name: 'items',
+            align: 'right',
+            label: 'Items',
+            field: row => (row.tag === 'tpos' && row.extra.details) || null
```

Comment:
why only `tpos`?

Reply:
only tpos has the details `extra` and it's the only extension until now to have item

Reply:
1. why exclude other/future ones?
2. core should not know/care about a particular extension

Reply:
should be just `field: row => (row.extra?.details) || null` ?

Reply:
or simply `field: row => row.extra?.details)` ?